src/vs/workbench/contrib/externalTerminal/electron-sandbox/externalTerminal.contribution.ts

```diff
@@ -107,7 +107,7 @@ export class ExternalTerminalContribution implements IWorkbenchContribution {
 					enumDescriptions: [
 						nls.localize('terminal.explorerKind.integrated', "Use VS Code's integrated terminal."),
```

Comment:
@Tyriar, should we be consistent and also use "Show the" for the other descriptions as well?